content/docs/getting-started/cms-set-up.md

```diff
@@ -56,9 +67,17 @@ export default App;
 
 ## Configure the CMS object
 
-When instantiating the CMS, you can pass [options](/docs/cms#cms-configuration) to configure how the CMS object works.
+The `tinacms` package provides two possible user interfaces: the sidebar and the toolbar.
+
+When instantiating the `TinaCMS` object, you can pass in [CMS options](/docs/cms#cms-configuration) to the CMS to configure the user interface (sidebar or toolbar), plugins, and APIs declaratively.
+
+> The [CMS options](/docs/cms#cms-configuration) are a **key part of constructing the CMS**. The `sidebar` property controls the _Sidebar UI_ — which renders forms, menus, and other UI that connect with content editing actions. It will not render unless the CMS is enabled.
+
+For this project, we are going to add the sidebar.
+
+### The Steps
```

Comment:
<img width="409" alt="Screen Shot 2020-08-18 at 10 46 24 AM" src="https://user-images.githubusercontent.com/36613477/90547300-27cb1a80-e140-11ea-93a5-8f449a53e196.png">

kind of unrelated but the same toc issue is pretty hectic with three instance of 'the steps'